src/tests/test_evidence_hunter.py

```
@@ -0,0 +1,53 @@
+    """ Load sampled claims from a JSON file.
+
+    Args:
+        path (str): The path to the JSON file containing the sampled claims.
+
+    Returns:
+        list[dict]: A list of dictionaries, each containing a sampled claim.
+    """
+    with open(path, 'r', encoding='utf-8') as f:
+        return json.load(f)
+
+def build_claim_objects(sampled_claims):
```

Ruff (0.11.9)

Missing docstring in public function

(D103)